components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/IdentityKeyStoreResolver.java

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
I believe the updated license header should be WSO2 LLC. WDYT?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/5794/commits/15925678b9c8c229e882eec214e5707d04c4170c